charts/rancher-monitoring/v0.1.5/values.yaml

```diff
@@ -172,7 +172,7 @@ exporter-node-windows:
   image:
     os: windows
     repository: rancher/wmi_exporter-package
-    tag: v0.0.3
+    tag: v0.0.4
```

Comment:
@luthermonson I can bump this to v0.0.5 but the manifest was never created (e.g. https://hub.docker.com/v2/repositories/rancher/wmi_exporter-package/tags/v0.0.5 does not exist)

Seems like this was because the 20H2 build failed (https://drone-publish.rancher.io/rancher/wmi_exporter-package/13/5/2); could you look into it?

If it's not a difficult fix, I'd ideally like to bump to the latest before pushing out 2 new charts.

Reply:
I'll approve, but want to see if `0.0.5` is possible.

Reply:
bumped